launch/launch/logging/__init__.py

```diff
@@ -407,25 +425,15 @@ def __init__(self, *args, **kwargs):
         self.propagate = False
 
 
-default_log_dir = _make_unique_log_dir(
-    base_path=os.path.join(os.path.expanduser('~'), '.ros/log')
-)
-
-
 def reset():
     """Reset logging."""
     # Reset existing logging infrastructure
     for logger in LaunchLogger.all_loggers:
         logger.setLevel(logging.NOTSET)
         del logger.handlers[:]
-    # Back to default logging setup
-    launch_config.log_dir = None
-    launch_config.file_handlers = {}
-    launch_config.screen_handler = None
-    launch_config(
-        level=logging.INFO, log_dir=default_log_dir,
-        log_format='default', screen_format='default'
-    )
+    launch_config.reset()
+    launch_config.configure(level=logging.INFO, log_dir=None,
```

Comment:
@clalancette `log_dir=None` has no effect here.

Reply:
Also, I wonder if `launch_config.reset()` shouldn't result in this configuration already.

Reply:
Hmm, and what would happen if someone calls `launch_config.reset()` instead of `launch.logging.reset()`?

Reply:
Good point.  I'm thinking that we should just remove the `configure` method completely, and just pass those parameters to `reset`.  That would alleviate both of your concerns above.  What do you think about that?

Reply:
Hm, I tried it out, and it got ugly.  So scratch that.

Instead, what about something like https://github.com/ros2/launch/blob/launch-logger-config-getters-setters/launch/launch/logging/__init__.py , where we switch to everything being a getter/setter?  Do you think that is better?

Reply:
That looks much better!

Reply:
All right, I'm going to merge that branch to this PR, and we can continue the review with that.